lib/site_encrypt/phoenix.ex

```diff
@@ -45,10 +45,21 @@ defmodule SiteEncrypt.Phoenix do
   """
   defmacro configure_https(config, https_opts \\ []) do
     quote bind_quoted: [config: config, https_opts: https_opts] do
+      # Default to cowboy
+      adapter = Keyword.get(config, :adapter, :cowboy)
```

Comment:
Shouldn't the default be `Phoenix.Endpoint.Cowboy2Adapter` as at https://github.com/phoenixframework/phoenix/blob/105d62677fe7198b3277c4b7fd12b8c21d5f3905/lib/phoenix/endpoint/supervisor.ex#L127?

Reply:
Ah, I didn't see that :+1: